themes/digital.gov/src/scss/new/_landings.scss

```diff
@@ -0,0 +1,93 @@
+@use "uswds-core" as *;
+
+.dg-landing {
+  &__hero {
+    position: relative;
+    width: 100%;
+
+    .dg-img-flexible {
+      filter: brightness(30%);
+      height: 30rem;
+      object-fit: cover;
+      width: 100%;
+    }
+
+    &-text {
+      margin-left: 15%;
+      position: absolute;
+      top: 20%;
+      width: 70%;
+
+      a {
+        margin-top: units(5);
+      }
+
+      h1 {
+        color: color("base-lightest");
+        font-size: 2.5em;
+        font-weight: font-weight("heavy");
+
+        @include at-media("tablet-lg") {
+          font-size: 4em;
+        }
+      }
+    }
```

Comment:
Are these `h1` styles necessary? Alternatively we could use a token size like `xl` or `2xl`.

https://designsystem.digital.gov/design-tokens/typesetting/font-size/